RevitClasher/Tools.cs

```diff
@@ -100,25 +95,26 @@ public static List<ClashItems> clashingElements(Document doc, Application app, D
                     LogicalOrFilter logicalOrFilter = new LogicalOrFilter(FormTools.SelectedHostCategories);
 
                     FilteredElementCollector bbClashingCollector = new FilteredElementCollector(doc, doc.ActiveView.Id).WherePasses(logicalOrFilter).WherePasses(bbFilter);
-                    foreach (var element in bbClashingCollector.ToElements())
+
+                    foreach (Element elementA in bbClashingCollector.ToElements())
```

Comment:
Renaming variable. `elementA`, indicates that this element is the one that will be used to determinate if there exists a collision or clash to a second element. (Clash are something like: element A against element B).